.gitignore

```diff
@@ -120,8 +120,8 @@ zz_generated_*_test.go
 # make-related metadata
 /.make/
 
-# Just in time generated data in the source, should never be committed
-/test/e2e/generated/bindata.go
+# Need to commit so it can be vendored into origin
```

Comment:
You can put this inside the test/e2e/generated dir as:

```
!bindata.go
```

and that will prevent you having to merge this file if the contents change.

Reply:
Done